django-stubs/template/context.pyi

```diff
@@ -55,7 +55,7 @@ class Context(BaseContext):
     ) -> None: ...
     @contextmanager
     def bind_template(self, template: Template) -> Iterator[None]: ...
-    def update(self, other_dict: dict[str, Any] | Context) -> ContextDict: ...
+    def update(self, other_dict: dict[_ContextKeys, Any] | Context) -> ContextDict: ...
```

Comment:
`error: Argument 1 to "update" of "Context" has incompatible type "Dict[str, Any]"; expected "Union[Dict[Union[int, str, Node], Any], Context]"  [arg-type]`

Need to think about some of this more. I'm currently seeing this error, which I don't currently understand.